core/management/commands/install_xapian.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+# Originates from https://gist.github.com/jorgecarleitao/ab6246c86c936b9c55fd
+# first argument of the script is Xapian version (e.g. 1.2.19)
+VERSION=$1
+
+# Cleanup env vars for auto discovery mechanism
+export CPATH=
+export LIBRARY_PATH=
+export CFLAGS=
+export LDFLAGS=
+export CCFLAGS=
+export CXXFLAGS=
+export CPPFLAGS=
+
+# prepare
+rm -rf $VIRTUAL_ENV/packages
```

Comment:
```suggestion
rm -rf "$VIRTUAL_ENV/packages"
```
`shellcheck` se plaint là, et à plein d'autres endroits aussi. Ça serait mieux, au cas où ton `VIRTUAL_ENV` contienne des espaces ou autre caractère louche.

Reply:
Je conaissaisp pas le tool, je l'ai mis dans mon éditeur
Je corrige tout ça :)

Reply:
Nice :-)